backend/src/akvo/lumen/component/emailer.clj

```diff
@@ -2,6 +2,8 @@
   (:require [cheshire.core :as json]
             [clj-http.client :as client]
             [akvo.lumen.protocols :as p]
+            [akvo.lumen.specs.components :refer (integrant-key)]
```

Comment:
Since we used vectors most of the places, is there a reason for using the list notation?